.github/workflows/buf-ci.yaml

```diff
@@ -26,4 +26,5 @@ jobs:
           username: ${{ secrets.BUF_USER }}
           token: ${{ secrets.BUF_TOKEN }}
           push_create_visibility: public
+          push_git_metadata: false
```

Comment:
This also pushes labels for tags/branches, so if we disable we should probably ensure this only runs on the main branch for pushes?